### ISSUE TITLE
Cannot sign out of application when using oauth

### ISSUE THREAD

**Issue by desa** [influxdata/chronograf#5333]
![save-fail](https://user-images.githubusercontent.com/1477539/70549637-62989f80-1b29-11ea-8c85-9e60ee30499c.gif)

**Comment by desa** [comment]
Steps to reproduce

```sh
docker run -e GH_CLIENT_ID=<github client id>-e GH_CLIENT_SECRET=<github client secret> -e TOKEN_SECRET=foo -p 8888:8888 quay.io/influxdb/chronograf
```

**Comment by andoks** [comment]
I have this problem on 1.7.14

**Comment by hoorayimhelping** [comment]
logs when trying this locally:

```
time="2019-12-27T21:45:45Z" level=error msg="Invalid principal" component="token_auth" method=GET remote_addr="172.17.0.1:44478" url=/chronograf/v1/me 
time="2019-12-27T21:45:45Z" level=info msg="Response: Forbidden" component=server method=GET remote_addr="172.17.0.1:44478" response_time="143.7µs" status=403 
time="2019-12-27T21:45:45Z" level=info msg="Response: OK" component=server method=GET remote_addr="172.17.0.1:44480" response_time=45.5115ms status=200 
time="2019-12-27T21:45:45Z" level=info msg="Response: Temporary Redirect" component=server method=GET remote_addr="172.17.0.1:44478" response_time="155.1µs" status=307 
time="2019-12-27T21:46:19Z" level=error msg="Unable to exchange code for token oauth2: server response missing access_token" component=auth method=GET remote_addr="172.17.0.1:44478" url=/oauth/github/callback?code=1c5348774b6771fcd115&state=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1Nzc0ODM3NDUsImlhdCI6MTU3NzQ4MzE0NSwibmJmIjoxNTc3NDgzMTQ1LCJzdWIiOiJSNGJhdFpvZnNXSjY5bDFsUFZLdjc2R0pMRkFPdTFXeGYxK1JQeWgzYWF3PSJ9.9H13cvf40IL5sb7ryeXzvsSY_IvpWmwjwjNngShb39U
```

Seems the key issue is here:

```Unable to exchange code for token oauth2: server response missing access_token```

**Comment by hoorayimhelping** [comment]
this has been fixed with the release of [1.7.17](https://github.com/influxdata/chronograf/releases/tag/1.7.17)
